.ci/FILE_HEADER

```diff
@@ -0,0 +1,2 @@
+Copyright 2022 MosaicML Composer authors
+SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: newline
```suggestion
SPDX-License-Identifier: Apache-2.0

```

Reply:
I dont think this is needed -- the previous header did not have a new line, and testing on a blank file, looks like the pre-commit works without.